setup/upgrade_major.rst

```diff
@@ -30,12 +30,14 @@ backwards incompatible changes. To accomplish this, the "old" (e.g. functions,
 classes, etc) code still works, but is marked as *deprecated*, indicating that
 it will be removed/changed in the future and that you should stop using it.
 
-When the major version is released (e.g. 4.1.0), all deprecated features and
+When the major version is released (e.g. 5.0.0), all deprecated features and
 functionality are removed. So, as long as you've updated your code to stop
 using these deprecated features in the last version before the major (e.g.
-3.4.*), you should be able to upgrade without a problem.
+``4.4.*``), you should be able to upgrade without a problem. That means that
+you should first :doc:`upgrade to the last minor version</setup/upgrade_minor>`
```

Comment:
If I'm very strict, there should be a whitespace between the anchor label and reference
```suggestion
you should first :doc:`upgrade to the last minor version </setup/upgrade_minor>`
```